vendors/materialize/css/materialize.css

```diff
@@ -1020,11 +1020,11 @@
 }
 
 .teal.lighten-1 {
-  background-color: #424242 !important;
+  background-color: #26a69a !important;
```

Comment:
I double checked everything in the pages and they look the same. I just saw that too. This is really strange.